components/services/ipfs/ipfs_service_impl.cc

```diff
@@ -204,89 +137,45 @@ void IpfsServiceImpl::Launch(mojom::IpfsConfigPtr config,
     return;
   }
 
-#if defined(OS_POSIX)
-  // Setup pipe hack here to monitor the daemon process and exclude other child
-  // processes created above.
-  SetupPipeHack();
-#endif
+  child_monitor_.reset(new brave::ChildMonitor());
```

Comment:
`std::make_unique`

Reply:
fixed in 76857501ecbfb2f31ff1023ec197de6d90d62618